app/models/course/assessment/submission.rb

```diff
@@ -181,6 +181,16 @@ def current_answers
     answers.select(&:current_answer?).group_by(&:question_id).map { |pair| pair[1].first }
   end
 
+  # Loads the answer ids of the past answers of each question
+  def answer_history
+    answers.unscope(:order).order(created_at: :desc).pluck(:question_id, :id, :current_answer).group_by(&:first).map do |pair|
```

Comment:
Let's have a scope in `answer.rb` to exclude `current_answers`, rather than filtering it here?

Reply:
Will need to have the `current_answers` when i'm using `pluck` first though, otherwise when there are no `non_current_answers` then i would have an empty array